app/models/complete_moab.rb

```diff
@@ -27,9 +27,7 @@ class CompleteMoab < ApplicationRecord
   belongs_to :moab_storage_root, inverse_of: :complete_moabs
   belongs_to :from_moab_storage_root, class_name: 'MoabStorageRoot', optional: true
 
-  # NOTE: we'd like to check if there is a different complete_moab for the preserved_object and
-  #  assign the other complete_moab to preserved_objects_primary_moab
-  has_one :preserved_objects_primary_moab, dependent: :destroy
+  has_one :preserved_objects_primary_moab, dependent: :destroy # to be removed
```

Comment:
to be removed for issue #1978 (removing primary moab database table and associations)